tealiumlibrary/src/main/java/com/tealium/core/messaging/Listener.kt

```diff
@@ -59,7 +60,7 @@ interface DispatchDroppedListener : Listener {
  * Executes when a Remote Command execution is required from an external module.
  */
 interface RemoteCommandListener : Listener {
-    fun onRemoteCommandSend(url: String)
+    fun onRemoteCommandSend(handler: RemoteCommand.ResponseHandler, url: String)
```

Comment:
similar thought; does it make sense to have a single argument for this? and could it therefore be used by the JSON RC functionality as well?